SpatialGDK/Source/SpatialGDK/Private/Interop/WellKnownEntitySystem.cpp

```diff
@@ -183,17 +311,22 @@ void WellKnownEntitySystem::MaybeClaimSnapshotPartition()
 		}
 	}
 
-	if (LocalServerWorkerEntityId == LowestEntityId && ServerCount >= NumberOfWorkers)
+	// If we're the new snapshot partition auth server
+	if (LocalServerWorkerEntityId == LowestEntityId)
```

Comment:
Hmm this logic will now run even if the correct number of servers aren't connected

Reply:
Yep, that's necessary for the snapshot partition auth server reelection.

Good point that we don't want to try it on startup. I think 99% of the time it's fine because the worker which'll win the leader election is likely the one to reach this point first (but we shouldn't rely on that).

Adding this after the `if (bSnapshotPartitionAuthServerCrashInProgress)` condition:
```
		else if (ServerCount < NumberOfWorkers)
		{
			// We're starting up a fresh deployment but not all servers have started yet.
			return;
		}
```